### PR TITLE
Update sl container for sl7 to current (build-date=20180904)

### PR DESCRIPTION
**Pull request by sl-team**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/sl_latest/Dockerfile b/sl_latest/Dockerfile
index cc92a24..27772fb 100644
--- a/sl_latest/Dockerfile
+++ b/sl_latest/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-7-docker.tar.xz /
 
-LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20180807"
+LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20180904"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_latest/sl-7-docker.tar.xz b/sl_latest/sl-7-docker.tar.xz
index 3b5c9fa..fe220e9 100644
Binary files a/sl_latest/sl-7-docker.tar.xz and b/sl_latest/sl-7-docker.tar.xz differ
```

</details>

**Comment by tianon** [comment]
Looks like this suffers from the same thing we noted in https://github.com/docker-library/official-images/pull/4608#issuecomment-406023313 (old tarballs in Git history).  Can you please rebase that branch to only have a single commit which touches the large tarballs to help save on cloning bandwidth/time/storage?

**Comment by jcpunk** [comment]
I'm only showing two commits in the SL7 branch now.

File: library/sl [hunk 1]
```diff
@@ -8,7 +8,7 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 1bc6a746a62e6ef13afc53aa585be9368fa3538b
+GitCommit: 9bf1a0c449251e6edf1f61d8f7a80a06e1d7fe13
```
Comment (tianon) [inline library/sl]: So this needs to be updated to `b1a2cb1c4a0efc1965f3344f8bdf1c949c7b79c2` now (https://github.com/scientificlinux/sl-docker/commits/7). :+1:

**Comment by jcpunk** [comment]
Somehow I closed this in the process....  I'll open a new PR...

**Pull request closed without merging**